bazel-haskell-deps.bzl

```diff
@@ -57,6 +57,7 @@ haskell_cabal_library(
     version = packages["lsp-types"].version,
     srcs = glob(["**"]),
     deps = packages["lsp-types"].deps,
+    haddock = False,
```

Comment:
What does this do? Is this related to adding M1 support?

Reply:
It disables building haddocks for that package. We already disable it for others. The haddocks build failed on M1, details are [in the commit message](https://github.com/digital-asset/daml/pull/13401/commits/e9092ea157bcc155c46005ae16319a2af813a07a).

Reply:
Nice, thanks for the explanation.